xikolo-ios/ViewControllers/Courses/CourseListViewController.swift

```diff
@@ -10,8 +10,25 @@ import UIKit
 
 class CourseListViewController : AbstractCourseListViewController {
 
+    @IBAction func segmentedControlChanged(sender: UISegmentedControl) {
+        switch sender.selectedSegmentIndex {
+        case 0:
+            showMyCoursesOnly(false)
+        case 1:
+            if UserProfileHelper.isLoggedIn() {
+                showMyCoursesOnly(true)
+            } else {
+                sender.selectedSegmentIndex = 0
+                performSegueWithIdentifier("ShowLoginForMyCourses", sender: sender) // maybe switch to My Courses after succesful login?
+            }
+        default:
+            showMyCoursesOnly(true)
```

Comment:
In our case, it should. But in theory, segmented controls can consist of more options, so Swift wants a default clause here. Is there a way to tell it we only have those two values here?